### ISSUE TITLE
Help Button is NOT labelled for accessibility readers

### ISSUE THREAD

**Issue by qqmmm** [Simon-Initiative/oli-torus#4427]
**Describe the bug**
The "Help" button is not labeled on the instructor view of the course home page. When I was using a screen reader to do the accessibility audit of the page, it would only read it as a "button" without saying what the button was for. 

**To Reproduce**
1. Go to 'Course home page', and find the question mark at the right corner 
It is located on every page of the course overview. 

**Expected behavior**
It should read it as "help button", or "click here for help"

**Screenshots**
![image](https://github.com/Simon-Initiative/oli-torus/assets/146996796/76856c29-86e0-4d1a-8f2c-b7edee88de6f)

**Environment (please complete the following information):**
 - Win
 - Chrome
 - Version: [e.g. 22]

**Additional context**
Add any other context about the problem here, for instance a full link to the page or resource if this problem is specific to a course or a resource in a course.
https://proton.oli.cmu.edu/sections/introduction_to_learning_engin_nu6db/instructor_dashboard/overview

**Comment by cjlim1092** [comment]
Note to team: this is filed by our accessibility intern who is doing an accessibility audit
